src/components/Apps/ChatAi/components/ChitChat/index.vue

```diff
@@ -204,6 +236,15 @@ export default {
     sendIntroduction(item) {
       this.showIntroduction = false
       this.onSendHandle(item.content)
+    },
+    insertCode(code) {
+      this.sendPostMessage({
+        name: 'INSERT_TERMINAL_CODE',
+        data: code.replace(/^[\s\r\n]+|[\s\r\n]+$/g, '')
+      })
+    },
+    sendPostMessage(data) {
+      window.parent.postMessage(data)
     }
   }
 }
```

Comment:
I'm sorry, but the provided code doesn't seem to be included here. Could you please re-post the part of the code that needs checking?